operations/reservations.md

```diff
@@ -49,6 +53,7 @@ Returns all reservations within scope of the Access Token, filtered according to
 | `ReservationGroupIds` | array of string | optional, max 1000 items | Unique identifiers of [Reservation groups](#reservation-group). |
 | `States` | array of string [Service order state](./productserviceorders.md#service-order-state) | optional | A list of service order states to filter by. |
 | `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Reservations](#reservation-ver-2023-06-06) were updated. |
+| `ScheduledStartUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Reservations](#reservation-ver-2023-06-06) scheduled start time. |
```

Comment:
```suggestion
| `ScheduledStartUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval filtering Reservations by their scheduled start time. |
```
Succinct wording suggested by Microsoft Copilot, and I cannot disagree with it. Also, over time I think we will need to drop the object references in descriptions, and so I'm going to suggest we stop including them in any updates.